en/Config/Connections.md

```diff
@@ -0,0 +1,60 @@
+---
+lastUpdated: 2018-10-09
+---
+
+# Connections
+
+This feature allows you to manage connections to external services. You can view, add, edit and delete the connections.
+
+Connections can be used to assets deployment. See [Deploy](../Deploy/index.md) for further information.
+
+## List
+
+You can view the list of the all connections your project have. Items descriptions are follows:
```

Comment:
You can view the list of the all connections your project has. Item descriptions are follows: